assignment/scripts/loops-practice.js

```diff
@@ -12,36 +12,73 @@ for (let i=0; i<4; i++) {
 
 // 1.a. TODO: Write a for loop to console.log the numbers from 0 to 5 
 //   - Which part of the example loop do you need to change to do this?
+for (let count=0; count <= 5; count ++){
+  console.log(count)
+}
 console.log('count from 0 to 5');
 
 // 1.b. TODO: Write a for loop to console.log the numbers from 3 to 5
 //   - Which part of the example loop do you need to change to do this?
+for (let count=3; count <= 5; count ++){
+  
+  console.log(count)
+}
+
 console.log('count from 3 to 5');
 
 // 1.c. TODO: Write a for loop to console.log EVEN numbers from 2 to 10
 //   - Which part of the example loop do you need to change to do this?
+let numberArray = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+let countEven = []; 
+for (let count = 0; count < numberArray.length; count ++){
+  if (numberArray[count] % 2 === 0){
```

Comment:
This works well if you don't know the numbers going in, because you are guaranteed to only log the evens. Be aware that you could have _also_ used `count += 2` after the last semicolon and just counted up numbers directly.